dumb-jump.el

```diff
@@ -664,7 +700,7 @@ or most optimal searcher."
            :regex "macro\\s*JJJ\\("
            :tests ("macro test(a)=1" " macro test(a,b)=1*8"))
 
-    (:type "variable" :supports ("ag" "rg") :language "julia" 
+    (:type "variable" :supports ("ag" "rg") :language "julia"
```

Comment:
Er, this was me habitually removing trailing whitespace when Emacs tells me to.

Reply:
No worries. It should be gone anyway. Thanks!